webpack.dev-server.ts

```diff
@@ -26,6 +26,7 @@ function createDevServer(): WebpackDevServer {
     },
     allowedHosts: "all",
     host: "localhost",
+    port: 9191,
```

Comment:
How about moving webpack-server port to common `vars.ts`, e.g.?

```ts
export const webpackDevServerPort: number = +process.env.WEBPACK_DEV_SERVER_PORT || 9191;
```

Reply:
I would have suggested letting it decide and then parsing out the value would be best but this also is a good idea.

Reply:
Move to common